src/OpenTelemetry.Exporter.ApplicationInsights/ApplicationInsightsTraceExporter.cs

```diff
@@ -548,6 +550,7 @@ private void SetHttpProperties(SpanData span, OperationTelemetry telemetry, stri
                 }
 
                 request.ResponseCode = resultCode;
+                request.Context.Operation.Name = request.Name;
```

Comment:
These values are non nullable?

Reply:
we always have span name and it's never null. we want to put span name into 2 different properties on the telemetry.